ci/playbooks/inherit_parent_scenario.yml

```diff
@@ -0,0 +1,29 @@
+# This allows to create a custom scenario file that would
+# just override some parts of an existing one.
+# For example, you create a DT based on the va-hci.yml.
+# You will then just add
+# `cifmw_parent_scenario: scenarios/reproducers/va-hci.yml`
+# in the file, and override/extend the scenario to match your
+# needs.
+# The parameter also supports a list of files if needed.
+#
+# File path are relative to the root of the ci-framework repository.
+- name: Inherit from parent parameter file if instructed
+  vars:
+    _file_list: >-
+      {% if cifmw_parent_scenario is defined and
+         cifmw_parent_scenario is string -%}
+         {{ [cifmw_parent_scenario] }}
+      {% elif cifmw_parent_scenario is defined and
+         cifmw_parent_scenario is not string and
+         cifmw_parent_scenario is iterable -%}
```

Comment:
A dict is iterable too :). Maybe you want to check `is not mapping`

Reply:
added, thanks!